src/libs/ui/components/list/list.tsx

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
 import styled from 'styled-components';
+import { MacScrollbar } from 'mac-scrollbar';
+
+import 'mac-scrollbar/dist/mac-scrollbar.css';
```

Comment:
Do we really need to import this style again? We already imported it in the popup layout